src/components/ProductCard/ProductCard.css

```diff
@@ -1,55 +1,52 @@
-.productCard {
+.product-card {
   width: 300px;
   position: relative;
   border-radius: 12px;
   overflow: hidden;
-  box-shadow: 0 4px 12px rgba(0, 0, 0, 0.1);
   background-color: #fff;
   transition: transform 0.3s ease;
   cursor: pointer;
   text-align: left;
 }
-.productCard:hover {
+
+.product-card:hover {
   transform: scale(1.03);
```

Comment:
We dont need this unless as it is not in the airbnb,unless  you just like the effect and oyu want to keep it:)